### PR TITLE
Update Cox Bid Adapter For 1.0+

### PR DESCRIPTION
**Pull request by reynold-cox**
## Type of change

- [ ] Bugfix
- [ ] Feature
- [x] New bidder adapter   
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Does this change affect user-facing APIs or examples documented on http://prebid.org?
- [ ] Other

## Description of change
Updating the Cox bid adapter to be 1.0+ compliant.

- test parameters for validating bids
```
{
  bidder: 'cox',
  params: {
    size: '300x250',
    id: 2000005991707,
    siteId: 2000100948180
  }
}
```

- reynold@coxds.com
- [x] official adapter submission

### CODE REVIEW

**Comment by mike-chowla** [comment]
I'm trying to figure out if the state variables in helper object are functioning as global variables. I think they are.  If the bidder gets instantiated twice, it looks to me that both instances of the bidder are going to refer to the same helper object.

Is that what is going to happen or am I getting this wrong?

**Comment by mike-chowla** [comment]
@reynold-cox if you could answer my questions bout helper object variables, that would get this moving along

**Comment by mike-chowla** [comment]
@reynold-cox to pass state between  buildRequests & interpretResponse you can additional fields to return object from buildRequests as that object will get passed back to interpretResponse

**Comment by mike-chowla** [comment]
Your server is sending back two Access-Control-Allow-Origin headers which is causing Chrome to reject the response

screen shot 2018-04-30 at 4 04 23 pm

screen shot 2018-04-30 at 4 05 07 pm

**Comment by mike-chowla** [comment]
I'm also seeing an error from getUserSyncs()

screen shot 2018-04-30 at 4 14 59 pm

screen shot 2018-04-30 at 4 15 12 pm

File: modules/coxBidAdapter.js [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+    }
+    if (flashVer > 4) return 15; else return 7;
+  };
+
+  let getRand = () => {
+    return Math.round(Math.random() * 100000000);
+  };
+
+  // State variables
+  let env = '';
```
Comment (mike-chowla) [inline modules/coxBidAdapter.js]: These variables are acting as global variables. 

Bidder Requirements:
_Support multiple instances: All adapters must support the creation of multiple concurrent instances. This means, for example, that adapters cannot rely on mutable global variables._

**Comment by reynold-cox** [comment]
Hi Mike

I'll do that and start a new PR tomorrow.   Thanks!

Reynold

**Comment by reynold-cox** [comment]
Sorry about the delay.  Thanks.

**Pull request merged**